api/anilist/src/main/kotlin/com/imashnake/animite/api/anilist/AnilistSearchRepository.kt

```diff
@@ -26,5 +23,6 @@ class ApolloSearchApi @Inject constructor(
                     search = Optional.presentIfNotNull(search)
                 )
             ).executeCacheAndNetwork()
+            .asResult { it.page!! }
```

Comment:
if you're still on this branch, can you do me a favour and put `.executeCacheAndNetwork()` on a new line? Cheers 😄

Reply:
You know you can do code suggestions right? 😛